src/main/java/nl/tudelft/contextproject/model/EntityState.java

```diff
@@ -1,19 +0,0 @@
-package nl.tudelft.contextproject.model;
-
-/**
- * Enum representing the state of an Entity.
- */
-public enum EntityState {
-	/**
-	 * The entity is dead and should be removed from the renderer.
-	 */
-	DEAD,
-	/**
-	 * The entity is alive and should receive updates.
-	 */
-	ALIVE,
-	/**
-	 * The entity is new and should be added to the renderer.
-	 */
-	NEW 
```

Comment:
I think the javadocs for the EntityState should be readded, as they provide additional information about what each state implies (should be removed, should receive updates and should be added).